### ISSUE TITLE
.md files

### ISSUE THREAD

**Issue by dt215git** [mampfes/hacs_waste_collection_schedule#503]
@mampfes: I've spent a couple of days trying to get a better understanding of GitHub's markdown. I used this project's README as a starting point because there was a reasonable amount of content, images, code examples, etc. Far easier to do that, that try to come up with something myself :grin: 

Most of the content is the same, but has been restructured as I've played around with things. You can see the various .md files [here](https://github.com/dt215git/hacs_waste_collection_schedule/tree/doc_updates/markdowns).

I was just wondering if there's anything in there you think might be useful in the main repo? If so, I can sanity check spellings, cross-links, replace the lorem ipsum text, etc, and make a PR with the changes. Not a problem if you don't, it was primarily a learning exercise for me rather than something planned as a PR.

**Comment by mampfes** [comment]
Wow, I'm deeply impressed. I like the lean style, much better than the current, bulky style.

The only comment that I have is that I would put the new files in the `doc` folder.

Thanks!!!

**Comment by dt215git** [comment]
The `markdown` folder was just so I could keep things separate rather than accidentally overwrite the originals. It's easy enough to move them to the `doc`  folder for a PR. I'd also be tempted to move all the `.png` files from the `doc` folder to the `images` folder as folder names and contents would then be more consistent.

There's a lot of duplication between the current `README` and `info.md` files, is there a reason both are needed?

**Comment by mampfes** [comment]
`info.md` is the entry point for [HACS](https://hacs.xyz/) - if you search for integrations within Home Assistant using the HACS UI. 

I had to create a separate file next to README.md because the HACS markdown renderer didn't support the relative links to the sources - at least when I started the project. Maybe this has been improved meanwhile.

Ideally I would like to have only one entry point (= markdown file) which is used for HACS and also for github. HACS can be configured to use either README.md or info.md.

**Comment by dt215git** [comment]
Certainly seems sensible to have HACS use the README.md file if it can be made to render correctly. Is the only way to test the  rendering to update the hacs.json to include `render_readme: true` and  see how it looks in HA?

In the meantime, I've made a shorter info.md file that lists the countries/providers, but doesn't link out to the individual pages. I've also cleaned up the other .md files so the cross-references work (hopefully haven't missed any!) and removed the lorem ipsum, etc. The original files have been relocated to an md_archive folder just in case there's a need to revert to them if any issues with the new ones become apparent. 

The Code of Conduct section and the Development Roadmap sections have been commented out of the README. A Code of Conduct might be good practice for open source projects, but not having one doesn't seem to have been an issue for you so far. Whether to include a Development Roadmap section is really your call given this is your project. 

One other thing that comes to mind ahead of making a PR - the three PR currently pending will have readme and info.md files of the old format. I could submit the PR, and they are asked to re-edit their branches to the new format. Or, I wait until they've been merged, and then quickly add those entries into the new format I have, and then generate the PR. How would you prefer to work around that?

**Comment by mampfes** [comment]
> Is the only way to test the rendering to update the hacs.json to include render_readme: true and see how it looks in HA?

This is the only way I know.

Code of Conduct whould be nice - but to be honest - I have never though about it. Development Roadmap is difficult. I don't have major plans for the integration. Therefore I would also leave it out for the moment.

There are always pending PRs. I merged everything that was possible (it was only #518) just now. And I would stop merging further PRs until we have finished your PR (if this is OK for you). Either we or the PR submitters than have to adapt their docu - I think that is the only way to go.
